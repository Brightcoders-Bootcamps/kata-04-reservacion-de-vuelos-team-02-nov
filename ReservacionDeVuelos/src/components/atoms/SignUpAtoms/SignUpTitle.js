import React from 'react';
import {Text} from 'react-native';
import {styles} from '../../../styles/SignUpStyles';

const SignUpTitle = () => {
  return <Text style={styles.title}>Sign Up</Text>;
};

export default SignUpTitle;
