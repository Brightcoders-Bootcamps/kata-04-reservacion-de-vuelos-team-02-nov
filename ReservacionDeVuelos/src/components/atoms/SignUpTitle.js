import React, {useEffect} from 'react';
import {Text, Button, TextInput} from 'react-native';
import {styles} from '../../styles/SignUpStyles';

const SignUpTitle = () => {
  return <Text style={styles.title}>Sing Up</Text>;
};

export default SignUpTitle;
