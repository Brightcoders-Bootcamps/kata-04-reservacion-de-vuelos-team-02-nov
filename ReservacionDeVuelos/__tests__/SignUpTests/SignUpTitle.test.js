import React from 'react';
import SignUpTitle from '../../src/components/atoms/SignUpAtoms/SignUpTitle';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<SignUpTitle />).toJSON();
  expect(tree).toMatchSnapshot();
});
