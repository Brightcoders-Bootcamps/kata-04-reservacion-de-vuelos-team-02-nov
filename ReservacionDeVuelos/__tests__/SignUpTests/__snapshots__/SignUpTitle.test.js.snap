// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#2D02FF",
      "fontSize": 20,
      "fontWeight": "bold",
      "marginBottom": 10,
    }
  }
>
  Sign Up
</Text>
`;
